templates/cronjob.yaml

```diff
@@ -0,0 +1,202 @@
+{{- if .Values.garbageCollect.enabled }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ template "docker-registry.fullname" . }}-garbage-collector
+  labels:
+    app: {{ template "docker-registry.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
```

Comment:
Missing `metadata.namespace` (see https://github.com/twuni/docker-registry.helm/blob/main/templates/configmap.yaml#L5 for reference)